bpf/process/bpf_execve_event.c

```diff
@@ -152,6 +152,16 @@ binary_filter(void *ctx, struct msg_execve_event *event, void *filename)
 	return value ? *value : 0;
 }
 
+static inline __attribute__((always_inline)) __u32
+read_execve_shared_info(void *ctx, __u64 pid, struct msg_process *p)
```

Comment:
It doesn't seem that you use `struct msg_process *p` somewhere. Maybe remove that?

Reply:
Yes, I was planning to cache errors on p->flags, but then we can't predict really errors and since these are multiple stages of hooking execve, the proper way to catch all those errors is to have a map per entier pid so we can transfert the errors happing at stage X of execve to the last execve_send() event per pid... which means we need a bigger map and more memory to reflect if lot of pids are doing execve at same time...

So I opted now with:
* tg_execve_joined_info_map with less entries and not really core information , just extra enhanced info and if the LRU map is full between all the execve then so be it, we won't have that info , but the metrics on the map will show it.
* stats tg_execve_joined_info_map_stats() map with 2 entries counter of entries for metrics and errors counter there... and we see in the future... @jrfastab in case for metrics on error maps.